phpBB/bin/phpbbcli.php

```diff
@@ -59,6 +59,7 @@
 require($phpbb_root_path . 'includes/compatibility_globals.' . $phpEx);
 
 $user = $phpbb_container->get('user');
+$user->data['user_id'] = ANONYMOUS;
 $user->add_lang('acp/common');
```

Comment:
Ah, I saw that $user->ip was already set to an empty string and thought "no work necessary here" but I guess that makes more sense. :D